.github/workflows/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+<!--
+Thanks for contributing a pull request! Please ensure you have taken a look at
+the contribution guidelines: https://github.com/skops-dev/skops/blob/main/CONTRIBUTING.rst
+-->
```

Comment:
you could add an explicit note here about linting (black, isort, pre-commit hooks, etc) maybe?